CMakeLists.txt

```diff
@@ -572,47 +572,78 @@ if (USE_PYTHON)
 
         # set extra python module path in src directory, if case PYTHON_COPY if OFF
         set(PYTHON_EXTRA_MODULES_PATH "${CMAKE_SOURCE_DIR}/src/python")
+        set(PYTHON_WRAPPER_PATHS "")
         
         if (PYTHON_COPY)
             set(TARGET_PREFIX "${CMAKE_BINARY_DIR}")
             set(PYTHON_EXTRA_MODULES_PATH "${TARGET_PREFIX}/lib/flow123d")
+              
+              # copy python files to build tree 
+              # first get prefix (we support just one prefix)
+              execute_process(
+                  COMMAND python -c "import sys; print sys.prefix"
+                  OUTPUT_VARIABLE PY_PREFIX
+                  OUTPUT_STRIP_TRAILING_WHITESPACE)        
+              execute_process(
+                  COMMAND python -c "import sys; print sys.exec_prefix"
+                  OUTPUT_VARIABLE PY_EXEC_PREFIX
+                  OUTPUT_STRIP_TRAILING_WHITESPACE)        
+              if (NOT (${PY_PREFIX} STREQUAL ${PY_EXEC_PREFIX}) )
+                  message(WARNING "Python exec_prefix (${PY_EXEC_PREFIX}) different then prefix (${PY_PREFIX}). Not supported.")
+              endif()
+                            
+            # get all valid python system paths
+            execute_process(
+              COMMAND python -c "import sys; print ';'.join([p for p in sys.path if p])"
+              OUTPUT_VARIABLE PY_PATHS
+              OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-            string(REGEX MATCH "python[^/]*$" PYTHON_SUBDIR ${PYTHON_INCLUDE_DIRS})
-            string(REGEX MATCH "python[0-9]\\.[0-9]" PYTHON_SUBDIR_MINOR ${PYTHON_SUBDIR})
             
-            # copy python files to build tree 
-            # first get prefix (we support just one prefix)
+            # get all subfolders from paths so we can copy them during install
+            # some are located in lib some in lib64 some in local
```

Comment:
Better comment, give an example. If the code does what I think the comment suggest a different function. However the code is to cryptic.